hamocc/inventory_bgc.F90

```diff
@@ -101,8 +100,8 @@ SUBROUTINE INVENTORY_BGC(kpie,kpje,kpke,dlxp,dlyp,ddpo,omask,iogrp)
   real :: zsedlayto(nsedtra)       ! Sum : Sediment layer tracers
   real :: zburial(nsedtra)         ! Sum : Sediment burial tracers
   !--- oceanic tracers
-  real :: zocetratot(nocetra)      ! Sum : Ocean tracers
-  real :: zocetratoc(nocetra)      ! Mean concentration of ocean racers
+  real :: zocetratot(-1:nocetra)   ! Sum : Ocean tracers
```

Comment:
I am a bit surprised to see the `-1` here. Is it really needed? - any tracers to be calculated should be `>0` (and loops also start at 1, where these values are calculated)